docs/mp/grpc/server-side-services.adoc

```diff
@@ -117,17 +141,17 @@ The Helidon gRPC MP API determines a method type by its annotation, which should
 @io.helidon.microprofile.grpc.core.Bidirectional
 ----
 
-=== Request an Response Types
+==== Request and Response Types
 A gRPC service method typically takes a request parameter and returns a response value (streaming methods may take or return
-multiple requests or responses). In traditional gRPC Java the types used for the request and response values must be
+multiple requests or responses). In traditional gRPC Java, the types used for the request and response values must be
 Protobuf serializable classes but this is not the case with Helidon gRPC. Helidon supports
  xref:../../se/grpc/marshalling.adoc[pluggable Marshallers]
 and by default will support any Java primitive or Java `Serializable` as well as Protobuf types. Any type that can be marshalled
 by the built-in marshallers or custom supplied marshaller may be used as a request or response type.
 
-=== Unary Methods
+==== Unary Methods
 A unary gRPC method is the simplest type of service method. Typically a unary method takes a request value and returns a
-response value but this does not have to be the case, a unary method could just as easily take no request parameter and/or
+response value but this does not have to be the case. A unary method could just as easily take no request parameter and/or
```

Comment:
response value, but this is not always the case.

Reply:
Updated